crates/astria-bridge-withdrawer/build.rs

```diff
@@ -1,4 +1,32 @@
+use ethers::contract::Abigen;
```

Comment:
should we be using something from alloy instead?

Reply:
don't think alloy has abigen yet